lib/services/presets.go

```diff
@@ -204,6 +207,7 @@ func defaultAllowRules() map[string][]types.Rule {
 			types.NewRule(types.KindSAMLIdPServiceProvider, RW()),
 			types.NewRule(types.KindOktaImportRule, RW()),
 			types.NewRule(types.KindOktaAssignment, RW()),
+			types.NewRule(types.KindLock, RW()),
```

Comment:
This adds the ability for users with the `editor` default role to be able to work with locks. Other users will need the lock resource rules added.